crates/nu_plugin_polars/src/dataframe/command/core/to_lazy.rs

```diff
@@ -34,16 +37,28 @@ impl PluginCommand for ToLazyFrame {
     }
 
     fn examples(&self) -> Vec<Example> {
-        vec![Example {
-            description: "Takes a table and creates a lazyframe",
-            example: "[[a b];[1 2] [3 4]] | polars into-lazy",
-            result: None,
-        },
-        Example {
-            description: "Takes a table, creates a lazyframe, assigns column 'b' type str, displays the schema",
-            example: "[[a b];[1 2] [3 4]] | polars into-lazy --schema {b: str} | polars schema",
-            result: Some(Value::test_record(record! {"b" => Value::test_string("str")})),
-        },
+        vec![
+            Example {
+                description: "Takes a table and creates a lazyframe",
+                example: "[[a b];[1 2] [3 4]] | polars into-lazy",
+                result: None,
+            },
+            Example {
+                description: "Takes a table, creates a lazyframe, assigns column 'b' type str, displays the schema",
+                example: "[[a b];[1 2] [3 4]] | polars into-lazy --schema {b: str} | polars schema",
+                result: Some(Value::test_record(record! {"b" => Value::test_string("str")})),
+            },
+            Example {
+                description: "Use a predefined schama",
+                example: r#"let schema = {a: str, b: str}; [[a b]; [1 "foo"] [2 "bar"]] | polars into-lazy -s {a: str, b: str}"#,
```

Comment:
This example doesn't seem to use the predefined schema, right?

Reply:
Good catch, thanks!

Reply:
Do we have access to all plugin commands in the example testing? would it be worth showing a use of `into-schema` as well?